build/package/rpm/README.md

```diff
@@ -0,0 +1,65 @@
+# FDO Server RPM Installation Guide
+This document covers installing and verifying FDO Server RPM packages.
+The FDO Server RPM packaging creates 4 packages:
+- `go-fdo-server` - Main package with binary and shared components
+- `go-fdo-server-manufacturer` - Manufacturer server systemd service
+- `go-fdo-server-rendezvous` - Rendezvous server systemd service
+- `go-fdo-server-owner` - Owner server systemd service
+
+All these packages will be installed from Fedora Official repos or COPR. 
+
+## 1.Installation
+
+```bash
+#If installing from COPR first enable COPR repository.
+sudo dnf copr enable @fedora-iot/fedora-iot
+
+# Install packages
+sudo dnf install go-fdo-server go-fdo-server-manufacturer go-fdo-server-rendezvous go-fdo-server-owner
+```
+
+## 2.Configuration
+
+Each service can be configured via /etc/sysconfig/ configuration files.
+- `/etc/sysconfig/go-fdo-server-manufacturer`
+- `/etc/sysconfig/go-fdo-server-rendezvous`
+- `/etc/sysconfig/go-fdo-server-owner`
+
+Example configuration (`/etc/sysconfig/go-fdo-server-manufacturer`):
+```bash
+LISTEN_IP="0.0.0.0"
+LISTEN_PORT="8038"
+DATABASE_DSN="/var/lib/go-fdo-server-manufacturer/db.sqlite"
+DATABASE_PASSWORD="@PassW0rD$"
+MANUFACTURER_KEY="/etc/go-fdo-server/manufacturer.key"
+OWNER_CRT="/etc/go-fdo-server/owner.crt"
+DEVICE_CA_CRT="/etc/go-fdo-server/device-ca.crt"
+DEVICE_CA_KEY="/etc/go-fdo-server/device-ca.key"
+```
+
+If not specified in the configuration file, **certificates** are automatically generated in /etc/go-fdo-server/ when the service starts.
```

Comment:
Double checked the code and it turns out that the certificates will be automatically generated if the files do not *exist* when the service starts. And it turns out you *always* have to specify the certificate paths in the env file else the service will fail to start.

So I'd recommend clarifying the text a bit by changing:
`If not specified in the configuration file, certificates are automatically...` 
to something like 
`If the certificate files do not exist then they are automatically...`